example_notebooks/5_ALS_hd_demo.ipynb

```diff
@@ -0,0 +1,1657 @@
+{
```

Comment:
<p>Did you create the <code>6-mls-wheat</code> branch from the previous feature branch (instead of from <code>dev</code>) or why is this in here?    </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/3dgeo-heidelberg/helios/pull/303/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
That might have been a mistake, is this a big problem?

Reply:
would it be the best thing to delete the files from the 6_mls_wheat branch or would that cause problems when merging the branch to dev?

Reply:
No, it's not a problem. When merging, we will see that there is a merge conflict and then we simply choose whether to keep the file from the dev branch or from your feature branch :)